flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/SqlClientTest.java

```diff
@@ -197,4 +219,14 @@ public void testVerboseErrorMessage() {
             assertThat(output, containsString(error));
         }
     }
+
+    @Test
+    public void testExecuteSqlFile() throws IOException {
+        String[] args = new String[] {"-f", sqlFilePath};
+        SqlClient.main(args);
+        final URL url = getClass().getClassLoader().getResource("sql-client-help-command.out");
+        final String help = FileUtils.readFileUtf8(new File(url.getFile()));
+        // TODO: escape test to trigger test
```

Comment:
what's this?

Reply:
I am too tired to fix this test last night. So I just leave the comment to inform myself...